sogs/http.py

```diff
@@ -1,6 +1,7 @@
 # success codes:
 OK = 200
 CREATED = 201
+NO_CONTENT = 204
```

Comment:
I don't think we should use this; everywhere else in SOGS where we have nothing to return we're still returning application/json content of an empty dict (`{}`) with 200 OK, which allows us flexibility to add return values later without changing the status code.